### PR TITLE
V2/Fix abstract app tests

### PR DESCRIPTION
**Pull request by Buckram123**

### CODE REVIEW

File: framework/packages/abstract-app/Cargo.toml [hunk 1]
```diff
@@ -39,6 +39,5 @@ workspace-hack = { version = "0.1", path = "../../workspace-hack" }
 
 [dev-dependencies]
 cosmwasm-schema = { workspace = true }
-speculoos = { workspace = true }
```
Comment (Buckram123) [inline framework/packages/abstract-app/Cargo.toml:42]: Also did small sidequest, hehe

Comment (Kayanski) [reply framework/packages/abstract-app/Cargo.toml:42]: Don't we like speculoos ? @CyberHoward

**Review by Kayanski — APPROVED** [review]

File: framework/packages/abstract-app/Cargo.toml (hunk 1 above)
Comment (Buckram123) [reply framework/packages/abstract-app/Cargo.toml:42]: We discussed it already when you were gone, but there is no benefit for using it. It's also not maintained and no one uses it.

File: framework/packages/abstract-app/src/endpoints/query.rs [hunk 2]
```diff
@@ -55,11 +55,10 @@ impl<
 
     fn dapp_config(&self, deps: Deps) -> StdResult<AppConfigResponse> {
         let state = self.base_state.load(deps.storage)?;
-        let admin = self.admin.get(deps)?.unwrap();
         Ok(AppConfigResponse {
-            account: state.account.into_addr(),
+            account: state.account.addr().to_owned(),
```
Comment (CyberHoward) [inline framework/packages/abstract-app/src/endpoints/query.rs:59]:
```suggestion
            account: state.account.into_addr(),
```

**Review by CyberHoward — APPROVED** [review]
LGTM!

File: framework/packages/abstract-app/src/endpoints/query.rs (hunk 2 above)
Comment (Buckram123) [reply framework/packages/abstract-app/src/endpoints/query.rs]: Thank you

**Pull request merged**